src/video_emotion/extract_classifier.py

```diff
@@ -45,7 +62,37 @@ def classify_video(video_path, time_range=None):
         "surprise": surprise_sum / number_of_faces,
         "neutral": neutral_sum / number_of_faces
     }
+def find_emotions_cutoffs(emotions):
+
```

Comment:
This line should be removed

And I think you can type to "emotions" due to your description of it down below